src/app/services/hris/employee/employee-profile.service.ts

```diff
@@ -25,6 +25,7 @@ export class EmployeeProfileService {
               private employeeStore:Store<{employees:EmployeeState}>, 
               private cookieService: CookieService) { }
 
+  //Endpoint not working, ID expected.
```

Comment:
What's happening here?

Reply:
Call seems Redundant?
Endpoints that get the EmployeeProfile is Expecting an ID, email or we just retrieve all.